apps/app/src/components/Admin/UserGroup/UserGroupForm.tsx

```diff
@@ -116,7 +116,7 @@ export const UserGroupForm: FC<Props> = (props: Props) => {
             <button
               type="button"
               id="dropdownMenuButton"
-              data-toggle="dropdown"
+              data-bs-toggle="dropdown"
```

Comment:
bs4 以前の記法から bs5 の記法に変更することで解決しました

## 参考
> Breaking Data attributes for all JavaScript plugins are now namespaced to help distinguish Bootstrap functionality from third parties and your own code. For example, we use data-bs-toggle instead of data-toggle. 

https://getbootstrap.com/docs/5.3/migration/#javascript